webapp/channels/src/components/dot_menu/index.ts

```diff
@@ -123,7 +123,7 @@ function makeMapStateToProps() {
             isMobileView: getIsMobileView(state),
             timezone: getCurrentTimezone(state),
             isMilitaryTime,
-            canMove: canWrangler(state, channel.type, threadReplyCount),
+            canMove: canWrangler(state, channel?.type || 'O', threadReplyCount),
```

Comment:
Should this default to `O`? I would think we'd want to default to a blank string.

Reply:
The empty string is not a valid channel type. I guessed open was the most "default" one, but 0/5.

Reply:
This is also a case of what should we do when the channel is undefined. It seems like we're doing a lot to protect from crashing, but if the channel is undefined, is that not an error state? Maybe we should be crashing?